### PR TITLE
add feature flags to ansys-mechanical cli

### PR DESCRIPTION
**Pull request by koubaa**

### CODE REVIEW

**Review by dipinknair — APPROVED** [review]

File: src/ansys/mechanical/core/run.py [hunk 1]
```diff
@@ -102,6 +103,13 @@ def _run(args, env, check=False, display=False):
     type=int,
     help="Start mechanical in server mode with the given port number",
 )
+@click.option(
+    "--features",
+    type=str,
+    default=None,
```
Comment (dipinknair) [inline src/ansys/mechanical/core/run.py:109]: https://mechanical.docs.pyansys.com/version/stable/getting_started/running_mechanical.html , docs need update

**Review by dipinknair — CHANGES_REQUESTED** [review]

File: src/ansys/mechanical/core/run.py (hunk 1 above)
Comment (koubaa) [reply src/ansys/mechanical/core/run.py:109]: @dipinknair thanks. I think we should split out a separate section for running mechanical using ansys-mechanical. The title of that document is "launching Pymechanical" which isn't exactly right.

Comment (dipinknair) [reply src/ansys/mechanical/core/run.py:109]: Yes. I also like the idea of having separate section for `ansys-mechanical`

**Review by dipinknair — APPROVED** [review]

File: src/ansys/mechanical/core/run.py (hunk 1 above)
Comment (koubaa) [reply src/ansys/mechanical/core/run.py:206]: @dipinknair I filed an issue for having a separate doc section

**Review by dipinknair — APPROVED** [review]

**Pull request merged**